examples/spanner-interleave-subquery/README.md

```diff
@@ -0,0 +1,23 @@
+# spanner-interleave-subquery
+
+This example contains the benchmark code to examine how efficient we can query for Cloud Spanner interleaved tables with subqueries.
+
+## Prerequisite
+
+Create a Cloud Spanner database with [schema.sql](schema.sql).
+
+## How to run the benchmark
+
+Run the following command.
```

Comment:
These instructions assume the user has golang installed. what version?
It would be much better experience to provide a small docker file with a builder image that has go installed and a very minimal runtime image (e.g. [distroless](https://github.com/GoogleContainerTools/distroless)) to run the built executable.

https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
That's good point! We need Go 1.15 or higher, so I added that in description.